protocol/discord/types.go

```diff
@@ -1,26 +1,80 @@
 package discord
 
 import (
+	"errors"
 	"fmt"
+	"sync"
 
 	"github.com/status-im/status-go/protocol/protobuf"
 )
 
-type ErrorType uint
+type ErrorCodeType uint
 
 const (
-	NoError ErrorType = iota
-	Warning
-	Error
+	NoErrorType ErrorCodeType = iota
+	WarningType
+	ErrorType
+)
+
+const MaxTaskErrorItemsCount = 3
+
+var (
+	ErrNoChannelData  = errors.New("No channels to import messages from")
+	ErrNoMessageData  = errors.New("No messages to import")
+	ErrMarshalMessage = errors.New("Couldn't marshal discord message")
 )
 
 type MessageType string
 
 const (
-	MessageTypeDefault MessageType = "Default"
-	MessageTypeReply   MessageType = "Reply"
+	MessageTypeDefault       MessageType = "Default"
+	MessageTypeReply         MessageType = "Reply"
+	MessageTypeChannelPinned MessageType = "ChannelPinnedMessage"
+)
+
+type ImportTask uint
+
+const (
+	CommunityCreationTask ImportTask = iota
```

Comment:
Unless your first enum is and `Unknown` type eg `UnknownTask`, it is best practice to declare the first line like below:
```suggestion
	CommunityCreationTask ImportTask = iota + 1
```

This is because by default an uninitialised uint will have a `0` value, and therefore risk a collision with your first enum